src/dbally/collection.py

```diff
@@ -174,6 +180,7 @@ async def ask(self, question: str, dry_run: bool = False, return_natural_respons
             dry_run: if True, only generate the query without executing it
             return_natural_response: if True (and dry_run is False as natural response requires query results),
                                      the natural response will be included in the answer
+            llm_options: options to use for the LLM client.
```

Comment:
"options to use for the LLM client" adds almost no additional information beyond the name of the argument itself. I think it would be worth having a more detailed information, including how those options will interact with the LLM client's default options.

Reply:
I will correct this in PR with anthropic integration.